packages/website/src/components/modals/Modal.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import clsx from 'clsx';
+import styles from './Modal.module.css';
+import CloseIcon from '../icons/CloseIcon';
+
+interface ModalProps {
+  readonly header: string;
+  readonly children: JSX.Element | (JSX.Element | false)[];
+  readonly isOpen: boolean;
+  readonly onClose: () => void;
+}
+
+function Modal(props: ModalProps): JSX.Element {
```

Comment:
i don't mind using components, and definitly that could be usefull, i'm just unsure what stack/library we should use 

if we decide to use reach than we should just use

https://reach.tech/accordion - for options
https://reach.tech/checkbox - instead of my implementation
https://reach.tech/tooltip - for tooltips after copying

is this component library maintained, are there better choices? there is lots of questions :P

Reply:
Ha, yeah... I'm down to hold off of them for now and treat using them as a followup issue.